### ISSUE TITLE
Option to Globally Disable Auto-Wiring of Filters on Dashboards

### ISSUE THREAD

**Issue by ixipixi** [metabase/metabase#38835]
**Is your feature request related to a problem? Please describe.**
If you have a dashboard that is comprised of both filtered an unfiltered cards by design, the auto-wiring functionality can become a point of frustration. each time you edit the filter it will auto-wire to all of the cards and you have to select the toast to undo it every time.

**Describe the solution you'd like**
An option to disable auto-wiring for the instance.

**Comment by notrom** [comment]
To add a reproduction ...

1. Create a dashboard with 3 questions based on the Sample DB > Orders table
2. Add a date filter to the dashboard
3. Link the new filter to "Order.Created At" on the first question, other questions are auto wired
![image](https://github.com/metabase/metabase/assets/4504437/2e62121f-6dbc-4826-a086-4415ec6d82b8)

4. Remove that filter from 2 questions, only 1 question is wired to the filter
![image](https://github.com/metabase/metabase/assets/4504437/723337f0-1aca-4fc8-9652-ebb103707ebf)

5. Change the remaining filter field to "User.Birth Date", all questions are wired to the filter again, not what I wanted, I've already said I want no filters on 2 of them.
![image](https://github.com/metabase/metabase/assets/4504437/5d72d139-e815-4c11-93a2-32312b849734)

6. The toast would undo the auto wiring but it's only displayed for limited time (maybe 4 seconds), an if you've got a large dashboard with lots of questions it can be tricky to figure out what's changed.

The auto wiring is great, but seems a bit too aggressive in the case where filter is empty, if it's already selected it won't be changed.

**Comment by eaglexer** [comment]
I completely agree with everyone, I tried to find a switch in the admin settings to disable "Auto-connect filters", but unfortunately there is no such function yet. I hope that in the future it will appear. At the moment, editing a dashboard, especially with a large number of tabs and filters, can be very annoying due to the constant linking of filters and the main thing is to have time to disable them before the banner disappears.

**Comment by mngr** [comment]
We changed this to auto-connection happen only after the user clicks on the suggestion to do that [here](https://github.com/metabase/metabase/issues/43182)
